src/main.c

```diff
@@ -2452,6 +2452,8 @@ void handle_movement(double dt) {
         float m = dt * 1.0;
         g->ortho = glfwGetKey(g->window, CRAFT_KEY_ORTHO) ? 64 : 0;
         g->fov = glfwGetKey(g->window, CRAFT_KEY_ZOOM) ? 15 : 65;
+        if (glfwGetKey(g->window, CRAFT_KEY_CROUCH)) viewBob_offSet(0);
```

Comment:
Good use of descriptive variables here.  It makes it much easier to understand that this code correlates to a change in the view when the crouch key is pressed.